Rust/Rust.sublime-syntax

```diff
@@ -1393,6 +1398,33 @@ contexts:
           scope: punctuation.definition.string.end.rust
           pop: true
 
+  c-string:
+    - match: '(c)(")'
+      captures:
+        1: storage.type.string.rust
```

Comment:
Well, in Python we don't scope leading modifiers `string.quoted`, which is applied by `meta_scope` here.

Reply:
Python however seems to be unique with that regard. All others (C, C++, D, R) behave like this PR.